tcf_website/static/css/site/pages/landing.css

```
@@ -173,6 +173,129 @@
+
+.trending__stack {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-3);
+}
+
+.trending-card {
+  display: block;
+  height: 5rem;
+
+  background-color: var(--bg-elevated);
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)